frontend/packages/console-app/src/components/user-preferences/language/__tests__/LanguageDropdown.spec.tsx

```diff
@@ -40,7 +40,7 @@ jest.mock('../getLastLanguage', () => ({
 
 const usePreferredLanguageMock = usePreferredLanguage as jest.Mock;
 const getLastLanguageMock = getLastLanguage as jest.Mock;
-const preferredLanguageValue = 'ja';
+const preferredLanguageValue = 'ja-JP';
```

Comment:
In the bug it says... "I recommend we add the region subtag for Chinese, in the format zh-CN" If this is only for Chinese, and not Japanese or Korean, then is this change needed?